src/actions/wallet/L1/writeUnsafeDepositTransaction.test.ts

```diff
@@ -22,7 +32,7 @@ test('default', async () => {
         isCreation: false,
       },
       value: 0n,
-      toChain: base,
+      toChainId: base.id,
```

Comment:
this isn't working for some reason, it seems to want Chain passed as well, re GetChain in viem